README.md

```diff
@@ -56,10 +56,12 @@ The Azure Key Vault Provider offers two modes for accessing a Key Vault instance
           array:
             - |
               objectName: secret1
+              objectAlias: SECRET_1     # [OPTIONAL] object alias
```

Comment:
can we state the version after which this is available. - `# [OPTIONAL available for version > 0.0.4] object alias`